### PR TITLE
fix(ui): add new theme and enable light/dark mode

### PR DESCRIPTION
**Pull request by Fjandin**
feat: add new theme and enable light/dark mode

### CODE REVIEW

File: src/app/me/tickets/ticket.component.tsx [hunk 1]
```diff
@@ -87,125 +88,129 @@ export function Ticket({ ticket, refetch, selected, onSelect }: TicketProps) {
...
-                  }
-                  onConfirm={async () => {
-                    await createTicketTransfer()
-                    await refetch()
-                  }}>
+          <div className={"flex flex-1 flex-col justify-end"}>
+            <div className={"overflow-hidden whitespace-nowrap"}>
+              <span className={"font-bold"}>{conference?.name}</span>
+            </div>
+            <div className={"text-sm text-gray-500"}>
```
Comment (suuunly) [inline src/app/me/tickets/ticket.component.tsx:110]: this should probably be changed from `text-gray-500` to `text-muted`, no? :)

File: src/components/header.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ const Header = () => {
   return (
     <header className="flex min-h-12 items-center justify-between">
       <div>&nbsp;</div>
-      <div className="m-2 mr-6">
+      <div className="m-2 mr-6 dark:[&_*]:text-white">
```
Comment (suuunly) [inline src/components/header.tsx:7]: that's some advanced tailwind css stuff 😆 looking good 😊👍🏻

**Review by suuunly — CHANGES_REQUESTED** [review]
Looks really nice! 😍 
Only thing I noticed was the `text-gray-500` 😄

File: src/app/me/tickets/ticket.component.tsx (hunk 1 above)
Comment (Fjandin) [reply src/app/me/tickets/ticket.component.tsx]: text-muted made it unreadable in dark mode.

Comment (suuunly) [reply src/app/me/tickets/ticket.component.tsx]: That's why text-muted should automatically be adjusted for dark mode and light mode, via the css variables 😄

**Pull request merged**